src/style.css

```diff
@@ -0,0 +1,91 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir.ttc);
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+.search-box {
+  display: flex;
+  position: relative;
+  margin: 20px 0;
+}
+
+.icon {
+  position: absolute;
+  left: 26px;
+  top: 25px;
+}
+
+.searchInput {
+  display: inline-block;
+  width: 100%;
+  font-family: "Avenir", sans-serif;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  padding: 24px 25px 23px 62px;
+  font-size: 16px;
+  line-height: 21px;
+  color: #3d4e61;
```

Comment:
Add CSS variables for repeatable colors.

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)